app/controllers/custom/budgets/investments_controller.rb

```diff
@@ -121,7 +121,7 @@ def resource_name
 
       def investment_params
         attributes = [:heading_id, :tag_list, :organization_name, :location,
-                      :terms_of_service, :related_sdg_list,
+                      :terms_of_service, :related_sdg_list, :estimated_price,
```

Comment:
Sorry, I misread the code :sweat_smile:. My suggestion isn't possible because the methor returns `params.permit(...)`. What I mentioned is only possible when there's one method returning the array of attributes (we usually call it `allowed_params`). Since this isn't the case here, your original change is fine :+1:.